.changes/next-release/Bug Fix-1859b589-4651-4fa5-9545-317fceaff3f1.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "fixed Delve debugger installation for Go Lambdas on MacOS"
```

Comment:
```suggestion
	"description": "Delve debugger installation fails for Go Lambdas on MacOS"
```

Reply:
Wouldn't it be "no longer fails"?

Reply:
The changelog lists items as "Fix" or "Feature", etc. There is no need to repeat "fixed" in the description. And it's clear from the "Fix" label that the thing described, is being fixed.

This usually leads to clearer, less-awkward wording for these changelog items.

Reply:
Yeah that makes sense. Grammatically we should state _what_ was fixed then, so "failures" instead of "fails".

Reply:
Generally active voice ("fails") leads to less-awkward, more-concise sentence structure.

Reply:
For me the active voice is kind of awkward though because the actor is implied. Toolkits is the one fixing the bug. So the following sentence is a fragment defining what the bug is.

I changed it to `fails` though for consistency. Full sentences are probably more intuitive for people to write.